packages/types/types/test.tsx

```diff
@@ -193,3 +201,39 @@ const myPlaceholderImage = (
     blur
   />
 )
+ // $ExpectType Element
+const myAvatar = <Avatar
+  className={'className'}
+  title={'Title'}
+  subtitle={'Subtitle'}
+  src={'opossum.jpg'}
+  initials={'TS'}
+  size={100}
+/>
+// $ExpectType Element
+const myBanner = <Banner
+  header={'Header text'}
```

Comment:
n-b strings can just be `header="..."`

Reply:
Oh perfect, that's generally my preference but at a glance I saw the brackets being used for strings!

Reply:
Unless you mean literally `{'...'}`, which works too